seneca/smart_contract_user_libs/stdlib.py

```diff
@@ -1,8 +1,20 @@
 import pickle
+from datetime import datetime
```

Comment:
Can't put this in a smart contract unless we override datetime.datetime.now() due to both clock skew between delegates and the need to replay contracts to bootstrap into network.

Reply:
noted in latest version that we should overwrite it, also changed it so that all datetime modules we use import from this stdlib.